Tasks/Sergey Yashchanka/Task_6/functions.py

```diff
@@ -0,0 +1,60 @@
+# Задание:
+
+# 1. Написать рекурсивную функцию для вычисления суммы всех элементов вложенных (любая глубина) списков.
+# Пример списка (синтаксис Python): [1, 2, [2, 4, [[7, 8], 4, 6]]], сумма элементов - 34
+# 2. Написать рекурсивную функцию для вычисления n первых чисел Фибоначчи.
+# Примеры вызова:
+# fib(5) -> 0,1,1,2,3
+# fib(10) -> 0,1,1,2,3,5,8,13,21,34
+# 3. Task from class
+
+# ------------------------------------------------------------------------------------------------------------------ #
+
+# Решение:
+# 1
+any_list = [1, 2, [2, 4, [[7, 8], 4, 6]]]
+
+
+def summary(array, summa=0):
+    """используя необычную распаковку списка решил сделать эту задачу без использования цикла!!!
```

Comment:
basically any cycle could be replaced with a recursion, and that what was the task goal in the first place anyway